content/2017-munich/sponsor.md

```diff
@@ -125,14 +125,100 @@ prices and benefits:
       <td>
         Free attendee tickets
       </td>
-      <td class="benefit diamond">3</td>
-      <td class="benefit platinum">2</td>
-      <td class="benefit gold">1</td>
+      <td class="benefit diamond">6</td>
+      <td class="benefit platinum">4</td>
+      <td class="benefit gold">2</td>
       <td class="benefit silver">1</td>
     </tr>
   </tbody>
 </table>
 
+We also have some **special sponsorship levels** to support specific causes:
+
+<table class="table sponsorships-table">
```

Comment:
Special levels got their own table because they didn't fit into the regular level table anymore, space-wise.